README.md

```diff
@@ -147,6 +148,31 @@ spec:
   ovs: {}
 ```
 
+## Macvtap
+
+**Note:** This feature is **experimental**. Macvtap-cni is unstable and its API
+may change.
+
+The operator allows the administrator to deploy the
+[macvtap CNI plugin](https://github.com/kubevirt/macvtap-cni/), simply by
+adding `mavtap` attribute to `NetworkAddonsConfig`.
+
+```yaml
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  macvtap: {}
+```
+
+Macvtap-cni must be explicitly configured by the administrator, indicating the
+interfaces on top of which logical networks can be created.
+
+A simple example on how to do so, the user must deploy a `ConfigMap`, such as in [this example](https://github.com/kubevirt/macvtap-cni/blob/master/examples/macvtap-deviceplugin-config.yaml).
```

Comment:
Should we set this example in https://github.com/kubevirt/cluster-network-addons-operator/pull/346/files#diff-e1a65158257c165bb0c41bf08016e58bR2-R8 ?

Reply:
Currently we don't have any tests of functionality of operands. I really want to add at least basic sanity checks, but we need to setup test lane for it first. No need to add the test.